src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/IHtmlFragmentCache.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    /// <summary>
+    /// An implementation of this interface provides a service to cache html fragments.
+    /// One client of this service is <see cref="CacheTagHelper"/>.
+    /// </summary>
+    public interface IHtmlFragmentCache
+    {
+        /// <summary>
+        /// Evaluates the <see cref="IHtmlContent"/> to render and caches it.
```

Comment:
Agree w/ @sebastienros. We write summaries about the method and that thingy is consistently singular. So `Evaluates` and `caches` are consistent, correct, read nicely, are generally better together (than another alternative), and [some other words].